src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -0,0 +1,460 @@
+          const tokenInfo = tokensInfo.filter((token) => token.tokenId === rate.tokenId)
+          return (
+            <View style={styles.cardLineContainer} key={index}>
+              <View style={styles.cardLineLabel}>
+                <Text style={styles.cardLabelText}>{rate.label}</Text>
+                <TokenIcons
+                  tokensInfo={tokenInfo}
+                  size={IconSize.XXSMALL}
+                  showNetworkIcon={false}
+                />
+              </View>
+              <Text style={styles.cardLabelText}>
+                {t('earnFlow.poolInfoScreen.ratePercent', { rate: rate.percentage.toFixed(2) })}
+              </Text>
+            </View>
+          )
+        })}
+      </View>
+    </Card>
+  )
+}
+
+function TvlCard({
+  earnPosition,
+  infoIconPress,
+}: {
+  earnPosition: EarnPosition
+  infoIconPress: () => void
+}) {
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const { t } = useTranslation()
+  const tvl = earnPosition.dataProps.tvl
+  const priceUsd = earnPosition.priceUsd
+  const tvlInFiat = useDollarsToLocalAmount(
+    tvl ? new BigNumber(tvl).times(new BigNumber(priceUsd)) : null
+  )
+  const tvlString = useMemo(() => {
+    return `${localCurrencySymbol}${tvlInFiat ? formatValueToDisplay(tvlInFiat) : '--'}`
+  }, [localCurrencySymbol, tvlInFiat])
+
+  return (
+    <Card testID="TvlCard">
+      <View style={styles.cardLineContainer}>
+        <View style={styles.cardLineLabel}>
+          <Text numberOfLines={1} style={styles.cardTitleText}>
+            {t('earnFlow.poolInfoScreen.tvl')}
+          </Text>
+          <Touchable onPress={infoIconPress} borderRadius={24}>
+            <InfoIcon size={16} color={Colors.gray3} />
+          </Touchable>
+        </View>
+        <Text style={styles.cardTitleText}>{tvlString}</Text>
+      </View>
+    </Card>
+  )
+}
+
+function AgeCard({ ageOfPool, infoIconPress }: { ageOfPool: Date; infoIconPress: () => void }) {
+  const { t } = useTranslation()
+  const dateInterval: Duration = intervalToDuration({
+    start: ageOfPool,
+    end: new Date(),
+  })
+
+  return (
+    <Card testID="AgeCard">
+      <View style={styles.cardLineContainer}>
+        <View style={styles.cardLineLabel}>
+          <Text numberOfLines={1} style={styles.cardTitleText}>
+            {t('earnFlow.poolInfoScreen.ageOfPool')}
+          </Text>
+          <Touchable onPress={infoIconPress} borderRadius={24}>
+            <InfoIcon size={16} color={Colors.gray3} />
+          </Touchable>
+        </View>
+        <Text style={styles.cardTitleText}>{formattedAge(dateInterval)}</Text>
+      </View>
+    </Card>
+  )
+}
+
+function LearnMoreTouchable({
+  manageUrl,
+  providerName,
+}: {
+  manageUrl: string
+  providerName: string
+}) {
+  const { t } = useTranslation()
+  return (
+    <View style={styles.learnMoreContainer}>
+      <Touchable borderRadius={8} onPress={() => navigateToURI(manageUrl)}>
```

Comment:
add analytics event

Reply:
Added in 4772ff3a2005e2925ed8493d9f663db01192237e!